app/views/setup/settings/_cni_plugin.html.slim

```diff
@@ -0,0 +1,31 @@
+.panel.panel-default
+  .panel-heading
+    h3.panel-title
+      | CNI Plugin
+      | &nbsp;
+      i class='glyphicon glyphicon-info-sign' title="Software that Kubernetes uses to manage the containers."
+  .panel-body
+    p The choice of CNI plugin is completely transparent to end-users of the
+      cluster. Neither Kubernetes manifests nor container images need to be changed.
+
+    .form-group
+      = f.label :cni, "Choose the cni"
+      br
+        .btn-group.btn-group-toggle.cni-btn-group data-toggle="buttons"
+          = label_tag :cni_plugin, nil, class: "flannel btn btn-default #{'btn-primary active' if @cni == "flannel"}", data: { element: '.flannel-desc' }
+            = f.radio_button :cni_plugin, "flannel", checked: @cni == "flannel"
+            | flannel
+          = label_tag :cni_plugin, nil, class: "cilium btn btn-default #{'btn-primary active' if @cni == "cilium"}", data: { element: '.cilium-desc' }
+            = f.radio_button :cni_plugin, "cilium", checked: @cni == "cilium"
+            | cilium
+
+      .flannel-desc.collapse class="#{'in' if @cni == "flannel"}"
+        | <em>Flannel</em> (<strong>default</strong>) is a production-ready cni plugin, fully supported by SUSE.
+
+      .cilium-desc.collapse class="#{'in' if @cni == "cilium"}"
+        p <em>cilium</em> is open source software for providing and transparently securing
+          network connectivity and loadbalancing between application workloads such as
+          application containers or processes. Cilium operates at Layer 3/4 to provide
+          traditional networking and security services as well as Layer 7 to protect 
+          and secure use of modern application protocols such as HTTP, gRPC and Kafka.
+          Cilium is integrated into common orchestration frameworks such as Kubernetes and Mesos.
```

Comment:
Some feedback:

  * highlight this is a tech-preview thing. We can copy the message we use for CRI-O and adapt that
  * Drop the final line (`Cilium is integrated...`).
  * It is true that it offer layer 7 protection, but this is not yet working because we don't have the envoy integration working right now. Am I correct? If that's correct I would drop that until we enable that.

Reply:
@flavio yes that's true L7 isn't really ready, Thanks for reviewing.